pyproject.toml

```diff
@@ -134,6 +135,16 @@ cmd = [
 ]
 description = "Run tests only for tests/tests.py"
 
+[tool.pixi.feature.test.target.osx-arm64.tasks.test-simple]
+cmd = [
+  "pytest",
+  "-m",
+  "not (needs_s3 or needs_envmodules)", # needs_s3 is not supported on osx-arm64
+  "-k",
+  "not (test_conda_global or queue)", # queue is buggy on osx-arm64
```

Comment:
I pulled this and testing locally the queue input dryrun fails for me too (am on ARM tho). Weirdly when I ran the test snakefile directly with Snakemake with --dry-run, the workflow ran. So something funky here. Will investigate.